<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>615</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>821</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="openImgButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImgButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blurImgButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grayscaleImgButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Grayscale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cropImgButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flipImgButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mirrorImgButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mirror</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateImgButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reversecolorImgButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorbalanceImgButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Balance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brightnessImgButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Brightness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contrastImgButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saturationImgButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noiseImgButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Noise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detectedgeImgButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Edges</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deblurImgButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Deblur</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
